observed.py

```diff
@@ -156,9 +159,10 @@ def __init__(self, inst, method_name, identify_observed, weakref_info):
         self.identify_observed = identify_observed
         key, d = weakref_info
         self.inst = weakref.ref(inst, CleanupHandler(key, d))
-        self.method_name = method_name
-    
-    def __call__(self, observed_obj, *arg, **kw):
+        self.method_name: str = method_name
```

Comment:
Is this type annotation necessary? The parameter `method_name` in `__init__` already has a type.